.github/scripts/updateChangelog.js

```diff
@@ -6,12 +6,19 @@
  * @param {string} releaseDate the release date to be written to the changelog
  */
 function updateChangeLog(changelogString, version, releaseDate) {
-    const optionalBetaOrRc = '(-rc.d+)?(-(b|beta).d+)?';
-    const changelogVersionRegExp = new RegExp(
-      `\\[(development|develop|unreleased|${version})${optionalBetaOrRc}.*`,
-      'gi',
-    );
-    return changelogString.replace(changelogVersionRegExp, `[${version}] - ${releaseDate}`);
-  }
-  
-  module.exports.updateChangeLog = updateChangeLog;
+  const optionalBetaOrRc = '(-rc.d+)?(-(b|beta).d+)?';
+  const changelogVersionRegExp = new RegExp(
+    `\\[(development|develop|unreleased|${version})${optionalBetaOrRc}.*`,
+    'gi',
+  );
+
+  const lastReleaseVersion = changelogString.match(/\[(\d+.\d+.\d+)\]/)[1];
```

Comment:
Not sure if this is reliable enough - if we ever use a version in brackets in a changelog entry, this would break.

Maybe we could match the whole link?

```suggestion
  const lastReleaseVersion = changelogString.match(/\[(\d+.\d+.\d+)\]: https:\/\/github\.com\/bitmovin\/bitmovin-player-ui\/compare\//)[1];
```

Reply:
We are matching the last released version. This is present at the starting of the file, after Unreleased section.

Reply:
got confused, thought you were taking it from the comparison link, not the header 🙈